viewer/packages/data-providers/src/Image360Provider.ts

```diff
@@ -0,0 +1,9 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import { Image360Descriptor, Image360DescriptorProvider, Image360FileProvider } from './types';
+
+export interface Image360Provider<T> extends Image360DescriptorProvider<T>, Image360FileProvider {
+  get360ImageDescriptors(metadataFilter: T): Promise<Image360Descriptor[]>;
```

Comment:
This is the same method that is declared in `Image360DescriptorProvider`, right? Does it need to be redefined here?